packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.spec.ts

```diff
@@ -677,7 +677,7 @@ describe("DefaultVisualizers unit tests", () => {
 								},
 								allowedTypes: {
 									value:
-										"{ apple : com.fluidframework.leaf.string | com.fluidframework.leaf.number | com.fluidframework.leaf.handle, banana : com.fluidframework.leaf.string | com.fluidframework.leaf.number | com.fluidframework.leaf.handle, cherry : com.fluidframework.leaf.string | com.fluidframework.leaf.number | com.fluidframework.leaf.handle }",
+										"Record<string, com.fluidframework.leaf.string | com.fluidframework.leaf.number | com.fluidframework.leaf.handle>",
```

Comment:
What was here before was definitely wrong, but the new content also doesn't seem right. Since this corresponds to an unnamed Map node, should this not be something like "Map<..."?

Reply:
I was trying to keep the new code as close to the old code as I could in this change, and the old type did a bad job of describing the fields object (it was an object signature not mentioning all the other possible keys) so I made it a more accurate (and easier to implement) version of the same thing.

In the future, if we want to make this describe the schema, we can port it to actually print the schema here, but I think that should be a separate change as it would impact a lot more of the tests than this.